### PR TITLE
more pillow retry errors

### PR DESCRIPTION
**Pull request by czue**
individual errors linked in the commits

@esoergel / @snopoke / anyone

### CODE REVIEW

File: custom/intrahealth/__init__.py [hunk 1]
```diff
@@ -144,7 +144,7 @@ def _get_location(form):
         if not user_id:
             return None
         try:
-            user = CommCareUser.get(user_id)
+            user = CouchUser.wrap_correctly(CouchUser.get_db().get(user_id))
```
Comment (esoergel) [inline custom/intrahealth/__init__.py]: nit: [`CouchUser.get_by_user_id`](https://github.com/dimagi/commcare-hq/blob/4bfb18d4f2ba2588ecbd5a905c76331f916fec43/corehq/apps/users/models.py#L1161-L1177) does this.

**Comment by esoergel** [comment]
:heart: the cleanup!

File: custom/intrahealth/__init__.py (hunk 1 above)
Comment (czue) [inline custom/intrahealth/__init__.py]: ah i was looking for that, thanks

Comment (czue) [inline custom/intrahealth/__init__.py]: https://github.com/dimagi/commcare-hq/pull/12494/

**Pull request merged**